### PR TITLE
docs: updating to have common open source project files

### PR DESCRIPTION
**Pull request by idunbarh**
## Description

Updates to `README.md` and adding common files found in open source projects that are aligned to the OpenSSF projects.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+
```
Comment (jhoward-lm) [inline CODE_OF_CONDUCT.md:1]:
```suggestion
```

File: COPYRIGHT.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (jhoward-lm) [inline COPYRIGHT.md:1]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,65 @@
 # bomctl
-Format agnostic SBOM tooling
+
+__bomctl__ is format-agnostic Software Bill of Materials (SBOM) tooling, which is intended to bridge the gap between SBOM generation and SBOM analysis tools. It focuses on supporting more complex SBOM operations by being opinionated on only supporting the [NTIA minimum fields](https://www.ntia.doc.gov/files/ntia/publications/sbom_minimum_elements_report.pdf) or other fields supported by [protobom](https://github.com/bom-squad/protobom).
+
+__This is an experimental project under active development. We'd love feedback on the concept, scope, and architecture!__
```
Comment (jhoward-lm) [inline README.md:5]:
```suggestion
> [!NOTE]
> This is an experimental project under active development. We'd love feedback on the concept, scope, and architecture!
```

**Review by jhoward-lm — CHANGES_REQUESTED** [review]
Need to update `.gitignore` to whitelist these 4 files

File: README.md (hunk 3 above)
Comment (jhoward-lm) [reply README.md:5]: See [GitHub markdown alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

**Review by jhoward-lm — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,2 +1,66 @@
...
+
+> [!NOTE]
+> This is an experimental project under active development. We'd love feedback on the concept, scope, and architecture!
+
+## Features
+
+- Work with multiple SBOMs in tree structures (through external references)
+- Fetch and push SBOMs using HTTPS, OCI, and GIT protocols
+- Leverage a `.netrc` file to handle authentication
+- Manipulate SBOMs like `diff`, `split`, and `redact`
```
Comment (mfrystacky) [inline README.md:13]: nit: Manipulate SBOMs **with commands** like `diff`, `split`, and `redact`

File: README.md [hunk 5]
```diff
@@ -1,2 +1,66 @@
...
+- Manage SBOMs using a persistent database cache
+- Interface with OpenSSF projects and services like GUAC and Sigstore
```
Comment (mfrystacky) [inline README.md:15]: Links to GUAC and Sigstore?

File: README.md [hunk 6]
```diff
@@ -1,2 +1,66 @@
...
+
+- [#bomctl on OpenSSF Slack](https://openssf.slack.com/archives/C06ED5VB81W)
+- [OpenSSF Security Tooling Working Group Meeting](https://zoom-lfx.platform.linuxfoundation.org/meeting/94897563315?password=7f03d8e7-7bc9-454e-95bd-6e1e09cb3b0b) - Every two weeks on Friday, 8am Pacific
+- [SBOM Tooling Working Meeting](https://zoom-lfx.platform.linuxfoundation.org/meeting/92103679564?password=c351279a-5cec-44a4-ab5b-e4342da0e43f) - Every Monday, 2pm Pacific
+
+## Commands
+
+### Fetch (Implemented)
+
+Ability to download an SBOM from a variety of SBOM storage mechanisms:
```
Comment (mfrystacky) [inline README.md:27]: nit: Ability to retrieve an SBOM via several protocols:

- HTTP/S
- Git

and from various locations:

- Local Filesystem
- OCI

File: README.md [hunk 7]
```diff
@@ -1,2 +1,66 @@
...
+
+- HTTP/S
+- OCI
+- Git
+- Filesystem
+
+This includes following external references in an SBOM to other SBOMs and loading them into the persistent cache. If SBOMs are access controlled, a user's [.netrc](https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html) file to authenticate.
```
Comment (mfrystacky) [inline README.md:34]: This includes **recursive loading** of external references in an SBOM to other SBOMs and loading them into the persistent cache. If SBOMs are access controlled, a user's [.netrc](https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html) file **is used** to authenticate.

**Review by mfrystacky — COMMENTED** [review]
A few nitpicks and suggestions for improvements

**Review by mfrystacky — APPROVED** [review]
:shipit:

**Pull request merged**